app.js

```diff
@@ -28,3 +32,80 @@ function applyStyles() {
 mainBarItemsLists.forEach((list) => {
   list.addEventListener('click', applyStyles);
 });
+
+const projectsList = [];
+
+// Factory function
+function objFac(title, frames, images, paragraphs, languages) {
+  return {
+    title,
+    frames,
+    images,
+    paragraphs,
+    languages,
+  };
+}
+
+function createElements() {
+  const div = document.createElement('div');
+  const containerdiv = document.createElement('div');
+  const BtnDiv = document.createElement('div');
+  const seeLiveBtn = document.createElement('button');
+  const seeSourceBtn = document.createElement('button');
+  const popHeader = document.createElement('div');
+  const exitIcon = document.createElement('button');
+
+  div.setAttribute('class', 'popup');
+  containerdiv.setAttribute('class', 'popup-container');
+  popHeader.setAttribute('class', 'pop-header');
```

Comment:
- [ ] Kindly your modal is missing scrolling to show the whole modal. 
- [ ] Kindly fix the tags and buttons positions. in the desktop version.
- [ ] When I click the close button the whole page is not scrolling anymore.

![image](https://user-images.githubusercontent.com/35535130/156461629-f3ed6449-d3fe-4517-942f-ea97182dfa18.png)